tests/testthat/test_learners_classiflabelswitch.R

```diff
@@ -62,7 +62,9 @@ test_that("no labels are switched", {
       hps = hpars[[id]]
       if (!is.null(hps))
         lrn = setHyperPars(lrn, par.vals = hps)
-      err = holdout(lrn, task, split = 0.5, stratify = TRUE)$aggr[[1L]]
+      tmp = holdout(lrn, task, split = 0.5, stratify = TRUE)
```

Comment:
Well, it makes debugging easier because you just have to uncomment the one line below.